src/sections/dataset/dataset-files/files-table/zip-download-limit-message/ZipDownloadLimitMessage.tsx

```diff
@@ -48,8 +49,10 @@ export function ZipDownloadLimitMessage({ selectedFiles }: ZipDownloadLimitMessa
   )
 }
 
-function getFilesTotalSizeInBytes(files: File[]) {
-  return files.map((file) => file.size).reduce((bytes, size) => bytes + size.toBytes(), 0)
+function getFilesTotalSizeInBytes(files: (File | undefined)[]) {
```

Comment:
Why do you consider `undefined` now and not before?

Reply:
because before your review I wasn't counting with the selectAll for manual pagination, which means that I need to select all files which I don't know because the pagination only retrieves 10 files, so those files that I cannot know are saving their place in the fileSelection list with an undefined

then the fileSelection object can have File | undefined, so this function `getFilesTotalSizeInBytes` that is reading the fileSelection now needs to accept some undefined files, to get the totalSizeInBytes it just ignores the undefined files

in the future we'll get the totalFileSize using a different endpoint but for the moment it can only read the size of the files of the current page